config.yaml

```diff
@@ -0,0 +1,11 @@
+options:
+  cuda-version:
+    type: string
+    default: "8.0.61"
+    description: |
+      The version of CUDA to be intalled.
+  cuda-subversion:
```

Comment:
I think version and sub-version should be combined into one config param. There's no reason for them to be separate afaict - they are always used together.